tests/wpt/metadata/webgl/conformance-1.0.3/conformance/attribs/gl-vertexattribpointer.html.ini

```diff
@@ -1,3 +1,2 @@
 [gl-vertexattribpointer.html]
```

Comment:
I think you need to remove the whole section. The usual way of doing this is running `./mach update-wpt /path/to/log` IIRC. I'll run this through automation so you get a log from our builders.